dashboard/db/schema.rb

```diff
@@ -191,6 +191,20 @@
 
   add_index "games", ["intro_video_id"], name: "index_games_on_intro_video_id", using: :btree
 
+  create_table "hint_view_requests", force: :cascade do |t|
```

Comment:
Good call. I'd also appreciate confirmation from @ashercodeorg that this schema is what he had in mind.